thrift/admin.thrift

```diff
@@ -301,4 +328,32 @@ struct ResendReplicationTasksRequest {
   60: optional i64 (js.type = "Long") startVersion
   70: optional i64 (js.type = "Long") endEventID
   80: optional i64 (js.type = "Long") endVersion
-}
\ No newline at end of file
+}
+
+struct GetDynamicConfigRequest {
+  10: optional string configName
+  20: optional list<configStore.DynamicConfigFilter> filters
+}
+
+struct GetDynamicConfigResponse {
+  10: optional shared.DataBlob value
+}
+
+struct UpdateDynamicConfigRequest {
+  10: optional string configName
+  20: optional list<configStore.DynamicConfigValue> configValues
+}
+
+struct RestoreDynamicConfigRequest {
+  10: optional string configName
+  20: optional list<configStore.DynamicConfigFilter> filters
+}
+
+struct ListDynamicConfigRequest {
+  10: optional string configName
```

Comment:
do we still need this param given that it's functionality can be achieved by GetDynamicConfig now?